test/clients.go

```diff
@@ -57,14 +58,18 @@ import (
 type clients struct {
 	KubeClient kubernetes.Interface
 
-	PipelineClient          v1beta1.PipelineInterface
-	ClusterTaskClient       v1beta1.ClusterTaskInterface
-	TaskClient              v1beta1.TaskInterface
-	TaskRunClient           v1beta1.TaskRunInterface
-	PipelineRunClient       v1beta1.PipelineRunInterface
-	PipelineResourceClient  resourcev1alpha1.PipelineResourceInterface
-	RunClient               v1alpha1.RunInterface
-	ResolutionRequestclient resolutionv1alpha1.ResolutionRequestInterface
+	V1beta1PipelineClient    v1beta1.PipelineInterface
+	V1beta1ClusterTaskClient v1beta1.ClusterTaskInterface
+	V1beta1TaskClient        v1beta1.TaskInterface
+	V1beta1TaskRunClient     v1beta1.TaskRunInterface
+	V1beta1PipelineRunClient v1beta1.PipelineRunInterface
+	PipelineResourceClient   resourcev1alpha1.PipelineResourceInterface
+	RunClient                v1alpha1.RunInterface
```

Comment:
Let's keep the naming consistent, what about `V1alpha1RunClient` and `V1alpha1ResolutionRequestClient`

Reply:
An example: 

https://github.com/tektoncd/pipeline/blob/1b941d31d7a47c8710cc19bc3b161ce265aa96dc/test/retry_test.go#L50

https://github.com/tektoncd/pipeline/blob/1b941d31d7a47c8710cc19bc3b161ce265aa96dc/test/featureflags.go#L93-L103

Reply:
Thanks Xinru, changed in https://github.com/tektoncd/pipeline/pull/5701